host-spec/c06-consensus.tm

```diff
@@ -255,10 +324,8 @@
 
     where <math|AuthorityDirectory<rsup|\<cal-E\><rsub|n>>> is the set of
```

Comment:
how does a validator in an epoch obtain <math|AuthorityDirectory<rsup|\<cal-E\><rsub|n>>>? It is important to obtain this set from at latest  epoch E_{n-3}

Reply:
As these are not (any longer?) computed by the host, but by the runtime, I moved any of those details to the runtime spec. I will leave this comment till it is under review.

Reply:
Yes, this is all part of the update.